src/routes/+layout.svelte

```
@@ -0,0 +1,8 @@
+<script lang="ts">
+	import SvelteTheme from '$lib/SvelteTheme.svelte';
+	let { children } = $props();
+</script>
+
+<SvelteTheme forcedTheme={data?.forcedTheme}>
```

ESLint

[error] 'data' is not defined.

(no-undef)